packages/api/src/routes/ehr/athenahealth/index.ts

```diff
@@ -11,12 +10,7 @@ const routes = Router();
 
 routes.use("/patient", patient);
 routes.use("/chart", chart);
-routes.use(
-  "/medical/v1/patient/:id",
-  processPatientRoute,
-  patientAuthorization("query"),
-  medicalPatient
-);
+routes.use("/medical/v1/patient", processPatientRoute, medicalPatient);
```

Comment:
@thomasyopes this was using `patientAuthorization("query")` but it looks like it was a param. Just want to make sure this will still work accordingly